wp-content/themes/ngisweden/functions/admin_dashboard.php

```diff
@@ -59,37 +67,48 @@
   'exclude' => $ignore_pages
 ));
 $missing_pages = [];
+$found_pages = [];
 foreach($ngi_pages as $ngi_page){
   if(!in_array($ngi_page->ID, $menu_page_ids)){
     $missing_pages[] = $ngi_page;
   }
+  else{
+    $found_pages[] = $ngi_page;
+  }
 }
 
-// BAD EXCERPTS
+// BAD EXCERPTS VS GOOD
 // Nearly everything should have an excerpt!
 // Check that those set are not too long.
-$methods_missing_excerpt = [];
+
 $all_cpt_posts = get_posts(array(
   'post_type' => array('methods', 'bioinformatics', 'technologies'),
   'posts_per_page' => -1
 ));
 $cpts_missing_excerpt = [];
+$cpts_with_excerpt = [];
 $cpts_excerpt_too_long = [];
+$cpts_excerpt_just_right = [];
```

Comment:
I don't think that this is ever added to..?

Reply:
Oh, yes `$cpts_excerpt_too_long = [];` isn't used either 🤔

Reply:
Oh no, it is, sorry